src/content/general/architecture/capi/index.md

```diff
@@ -0,0 +1,223 @@
+---
+linkTitle: Cluster-API-Architecture
+title: The Giant Swarm Platform Architecture
+description: Architecture overview explaining how our Platform is built it.
+weight: 30
+menu:
+  main:
+    parent: general-architecture
+last_review_date: 2022-08-12
+user_questions:
+  - Do you run a Developer Platform?
+aliases:
+  - /basics/platform-architecture/
+owner:
+  - https://github.com/orgs/giantswarm/teams/team-rocket
+---
+
+The Giant Swarm Platform consists of various systems. They can be categorized into three areas: infrastructure, applications, and operations.
+
+For managing the infrastructure we run a management cluster per provider and region wherever you want to have your workloads. From that management cluster you can spin up as many individual Kubernetes clusters, called workload clusters, as you want. Our operations team works to maintain all cluster components healthy, while we release new versions with new features and patches. On top of that, Giant Swarm offers a curated catalog with the most common Cloud Native tools that helps to monitor, secure or manage your applications. Customers can leverage those while we carry the burden of maintain and keep them up to date.
+
+Giant Swarm's architecture is split into two logical parts. One comprehends the management cluster and all the components running there. On the other side we have the workload clusters that are created dynamically by the users and they serve to run their business workloads. In principle the management cluster and workload cluster are the equal in terms of infrastructure and configuration. The difference comes with the additional layers we deployed on top of the management cluster that helps to manage your users and permissions, workload clusters or the applications running on the workload clusters.
+
+## Cluster Architecture
+
+As explained previously, both the management cluster and the workload cluster have the same cluster structure and configuration. In Giant Swarm we rely on [Cluster API](https://cluster-api.sigs.k8s.io/) to bootstrap and configure the cluster infrastructure and set up all the components needed for a cluster to function.
+
+[Cluster Architecture Image](./CAPI_architecture.png)!
+
+By default, the machines are split into three different failure domains or zones to ensure the availability of the API and workloads running on the there. In our setup, three control plane machines hold the Kubernetes API and the other controllers, and a variable number of worker machines contain the regular services.
+
+There is a machine created as a bastion that helps us with the operations. It is the single entry point to the running infrastructure so that way all the cluster machines can live in a private network and expose the services running on them via explicit configuration.
+
+{{< tabs >}}
+{{< tab id="flags-aws" title="AWS">}}
+
+{{< /tab >}}
+{{< tab id="flags-capz" title="Azure">}}
+
+{{< /tab >}}
+{{< tab id="flags-gcp" title="GCP">}}
+
```

Comment:
@fiunchinho  can you help me to fill this part, how CAPG networking works in general?